chirpstack/src/storage/device.rs

```diff
@@ -216,6 +216,14 @@ pub struct Filters {
     pub search: Option<String>,
 }
 
+#[derive(Clone, Debug)]
```

Comment:
You might want to add `Default` to `derive(...)` and add `#[default]` above `Name`.